source/declarations.tex

```diff
@@ -3918,14 +3918,12 @@
 }
 \end{codeblock}
 
-\pnum
 The \tcode{carries_dependency} attribute on function \tcode{f} means that the
```

Comment:
Fully agreed. For this and a few other cases, I saw no way other than removing the number, but keeping the paragraph break.